insights/combiners/grub_conf.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, grub1, grub2, grub1_efi, grub2_efi, grub_bles,
             self.is_efi = self.is_efi if sys_firmware else self.grub._efi
         else:
             _grub1, _grub2 = (grub1_efi, grub2_efi) if self.is_efi else (grub1, grub2)
-            if rh_rel and rh_rel.rhel8:
+            if rh_rel and rh_rel.rhel8 and _grub2 is not None and 'blscfg' in _grub2.get('configs', ""):
```

Comment:
@ryan-blakley - Updated, please have a check.

Reply:
@wushiqinlou looks good, only one thing, could you remove RedHatRelease from the combiner since it's not used anymore? The test will probably have to be updated to remove it as well.

Reply:
@ryan-blakley - the removal of RedHatRelease will break the tests of rules and your Apps again. We'd better do it together when the related rules and Apps get ready if we must do that.

Reply:
@wushiqinlou Good point, I'm good with merging them together.